SwiftCompilerSources/Sources/SIL/Function.swift

```diff
@@ -262,3 +282,17 @@ extension BridgedArgumentConvention {
     }
   }
 }
+
+extension Int {
```

Comment:
Please no such extensions on `Int`.
Instead you can define (private) functions like: `func isDerived(flags: Int) -> Bool`